helm/cluster/templates/clusterapi/controlplane/_helpers_files.tpl

```diff
@@ -0,0 +1,9 @@
+{{- define "cluster.internal.controlPlane.kubeadm.files" }}
```

Comment:
Helper template for Files configuration in KubeadmControlPlane. It also includes those files that are common across both control plane and workers.